proxies/live/apiproxy/resources/jsc/ValidateTarget.js

```diff
@@ -0,0 +1,19 @@
+var b64decoded = JSON.parse(context.getVariable("targetIdentifierDecoded"));
```

Comment:
It feels a bit odd to me to be completing some of the target identifier validation in Javascript and some of it in Python, I think it would be simpler if all of this validation was either completed in Python or Javascript (perhaps Javascript as we're already using it elsewhere in the proxy whereas I think this would be the first usage of Python outside of the tests). Given time constraints it might be worth looking at this as a tech debt as this certainly works for the moment 🙂 .

Reply:
I think it is worth raising a tech debt for this. Would be nice to stick with js if we can, just to make it simpler.

Reply:
https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-61146

Reply:
Is this the right JIRA?

Reply:
No copy and pase errror
should be : https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-62112